simulator/mips/mips_instr.h

```diff
@@ -131,10 +131,21 @@ class BaseMIPSInstr
         MIPSRegister get_dst2_num() const { return dst2; }
         std::string_view get_disasm() const { return static_cast<std::string_view>( disasm); }
 
-        /* Checks if instruction can change PC in unusual way. */
-        bool is_jump() const { return operation == OUT_J_JUMP      ||
-                                      operation == OUT_R_JUMP      ||
-                                      operation == OUT_BRANCH;}
+	/* Checks if instruction can change PC in unusual way. */	
+
+	bool is_jump() const { return operation == OUT_J_JUMP ||
+				      operation == OUT_R_JUMP ||
+				      operation == OUT_BRANCH; }
+
+	//target is known at ID stage and always taken
+	bool is_direct_jump() const { return operation == OUT_J_JUMP; }
```

Comment:
Mostly probably, same methods has to be added to RISC-V instruction stub, as we still do not have a common ancestor class for MIPS and RISC-V instructions (#723)